src/features/game/lib/gameMachine.ts

```diff
@@ -123,14 +122,18 @@ export function startGame(authContext: AuthContext) {
               };
             }
 
-            // They are an anonymous user
-            // TODO: Load from Web3
+            // Visit farm 
+            const address = authContext.address as string;
+            const game = await getVisitState(address);
 
-            return { state: INITIAL_FARM };
+            game.id = authContext.farmId as number;
+
+            return { state: game };
+
+            // return { state: INITIAL_FARM };
```

Comment:
We also have the case where the `API_URL` environment variable is null. This is used for quickly running the repo and on the `https://sunflower-land.com/art-preview-only` URL.

You will want to consider how we can support those cases. Right now it throws an error

<img width="520" alt="Screen Shot 2022-02-15 at 2 24 38 pm" src="https://user-images.githubusercontent.com/11745561/154115812-e17fdc19-7ab1-40ac-bad2-d3bb177c6b04.png">

Reply:
Thanks for this one, did not notice 😅 

made changes in `gameMachine.ts` to handle preview mode i.e. returning `INITIAL_FARM`

![sfl-visit-farm-initital-farm](https://user-images.githubusercontent.com/89294757/154190947-d519a8a6-0290-4d6a-ae29-6496bf1b80b5.PNG)